plugins/hosts/linux/cap/os_arch.rb

```diff
@@ -0,0 +1,20 @@
+include Vagrant::Util
+
+module VagrantPlugins
```

Comment:
> qualify in order to download the right binary 

Didn't get by right binary?

Code to download binary is present in `lib/vagrant-service-manager/plugin_util.rb` line 147.